eoscostarica/guias.eoscostarica.io
# docs(frontend): add publish ppt presentations and videos 459

### Publish PPT presentations and Videos

### What does this PR do?

- resolves #459

### Steps to test
1. Go to Developer Guidelines.
2. Go to community.
3. See changes 

#### CheckList
